tests/test_in_out.py

```diff
@@ -232,3 +232,22 @@ def test_spinner_nested_hiding_with_context_manager(capsys):
     out, _ = capsys.readouterr()
     out = out.replace("\r\033[K", "")
     assert "{}\n{}".format(HIDDEN_START, HIDDEN_END) in out
+
+
+def test_spinner_nested_hiding_with_context_manager_and_exception(capsys):
```

Comment:
`capsys` is not used inside the test, let's remove it from the arguments. I'd also rename the test, because it is not testing the "nested" case.

```suggestion
def test_spinner_hiding_with_context_manager_and_exception():
```